data/starts.txt

```diff
@@ -221,3 +221,101 @@ substitutions
 	"<starting planet>" "New Boston"
 	"<starting planet>" "Greenwater"
 		has "hai space start"
+
+
+
+start "lampyrid"
+	name "Lampyrid Start"
+	description `You grew up on New Boston, a world in the Dirt Belt. You've dreamed of owning a starship ever since your first visit to the spaceport, and years later you managed to acquire the wreck of an old Lampyrid-class transport.`
+	description `After many long years of working at a textile mill during the day and working on your ship during the night, you've finally saved enough credits to apply for a pilot's license from the Republic and take to the skies in the old ship.`
+	thumbnail "scene/lobby"
+	date 16 5 3013
+	system "Rutilicus"
+	planet "New Boston"
```

Comment:
Might it be fun to have this start be somewhere else, like Luna?
Just a question.

Reply:
It might, but part of the point is to minimize changes. It's a small, subtle difference to the way the player starts; not a massive shift. 

Speaking of which, I need to tweak the intro mission with a branch to remove the bit about the shipyard owner.